app/views/hq/partners/index.html.erb

```diff
@@ -35,7 +35,7 @@
                 <%= partner.status.name %>
               </td>
               <td>
-                <%= partner.start_date.strftime('%B %d, %Y') %>
+                <%= partner.start_date.strftime('%d %B %Y') %>
```

Comment:
Also this helper shouldn't die if date is nil, and probably shouldn't be namespaced under Admin